src/amo/routes.js

```diff
@@ -14,6 +15,7 @@ export default (
     <IndexRoute component={Home} />
     <Route path="addon/:slug/" component={DetailPage} />
     <Route path="addon/:slug/review/:reviewId/" component={AddonReview} />
+    <Route path="categories/:addonType/" component={CategoriesPage} />
```

Comment:
I think most URLs on the existing site are `:addonType/:somethingSomething` so while I think categories first actually might make more sense, I'll go for consistency given it's a toss-up otherwise anyway.